CMakeLists.txt

```diff
@@ -41,6 +41,17 @@ endif()
 file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/_env/EXTERNAL_BUILD_ASSETS.txt" "${EXTERNAL_BUILD_ASSETS}")
 MESSAGE(STATUS "EXTERNAL_BUILD_ASSETS: ${EXTERNAL_BUILD_ASSETS}")
 
+set(GLES_OPTION OFF)
```

Comment:
could this be set(GLES_OPTION ENV{USE_GLES}) and then everything below could use GLES_OPTION?

Reply:
Since cmake assumes false if it cannot read a variable, this simplifies the code quite a bit.